build/templates/session.py.mako

```diff
@@ -18,6 +18,8 @@ ${encoding_tag}
     functions = helper.extract_codegen_functions(functions)
 
     session_context_manager = '_' + config['context_manager_name']['task'].title() if 'task' in config['context_manager_name'] else None
```

Comment:
I would put all these into an if statement so that if task exists, they all must since the definition of the "task' context manager will need initiate and abort, but if task doesn't exist, the others are not required. They are now.
``` python
session_context_manager = None
if 'task' in config['context_manager_name']:
    session_context_manager = '_' + config['context_manager_name']['task'].title()
    session_context_manager_initiate = '_' + config['context_manager_name']['initiate']
    session_context_manager_abort = '_' + config['context_manager_name']['abort']
```

Reply:
fixed